src/com/form-field.js

```diff
@@ -312,6 +314,17 @@
 			}
 		};
 
+		var isValidDocumentSize = function (value) {
+			//Validate max size
+			//2Mo -> Bytes
+			if (!!value) {
+				return parseInt(value) < 2 * 1024 * 1024;
```

Comment:
Should the 2mo be configurable ?

Reply:
In the future, yes. But I would like to merge the code _as is_ because it works.